client/stylus/main.styl

```diff
@@ -1,28 +1,30 @@
+@css {
+  @import url('https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;700&family=Rajdhani:wght@400;700&display=swap');
```

Comment:
There is a discussion to be had for if we intend to bundle font's in the project.

Reply:
Do you mean their usage in general or would this cause React to literally bundle the files as part of the production build?

Reply:
Both. If we request the fonts from a CDN, people running Ampache on a closed network won’t have the fonts.  There are also privacy concerns regarding Google fonts. 

The only benefit of using Google fonts is speed when loading out of the local network.